python_basics_lists_dictionaries/python_basics_lists_dictionaries.md

```diff
@@ -136,21 +136,36 @@ There are a few important characteristics of lists:
      </lia-keep>
      </div>
 
-   * `.remove()` removes a specific element by **value**.
+     The element at index `0` (the first position in the list) has now been removed from `produce`:
+
+     @sage
+     <div class="python_link">
+     <lia-keep>
+     <script type="text/x-sage">
+
+     print(produce)
+
+     </script>
+     </lia-keep>
+     </div>
+
+   * **`.remove()`** removes a specific element by **value**.
 
     @sage
     <div class="python_link">
     <lia-keep>
     <script type="text/x-sage">
 
-    produce.remove("apple")
+    produce.remove('apple')
 print(produce)
 
     </script>
     </lia-keep>
     </div>
 
-  * `del` deletes objects, and can be used as another way to remove elements of a list by index. The syntax will be a little different, because it is a [**keyword**](https://realpython.com/python-keywords/), not a list method. It is useful because it allows you to delete list elements in a **slice**. Slicing uses subsetting notation to indicate where the slice begins and ends. The first index listed is **inclusive**, or included in the slice; this is followed by a colon; the second index listed is **exclusive**, or excluded from the slice; finally, these numbers are surrounded by square brackets.
+    Note that the `.remove()` method, unlike `.pop()`, does not itself return any output. If you want to see what happens if you run `print(produce.remove('apple'))`, try changing the code block above (be sure to re-run all of the code blocks above it!).
+
+  * **`del`** deletes objects, and can be used as another way to remove elements of a list by index. The syntax will be a little different, because it is a [**keyword**](https://realpython.com/python-keywords/), not a list method. It is useful because it allows you to delete list elements in a **slice**. Slicing uses subsetting notation to indicate where the slice begins and ends. The first index listed is **inclusive**, or included in the slice; this is followed by a colon; the second index listed is **exclusive**, or excluded from the slice; finally, these numbers are surrounded by square brackets.
```

Comment:
I wonder if `del` should be in a "learn more" box or some other aside because it isn't a method and we really aren't teaching learners how to use it at all (at least not directly in this paragraph), just telling them that there are other ways to do the same thing... maybe even an "other options" box?

Reply:
I like this idea!